tests/conftest.py

```diff
@@ -3777,7 +3793,7 @@ def store_with_sequencing_metrics(
     helpers: StoreHelpers,
 ) -> Store:
     """Return a store with multiple samples with sample lane sequencing metrics."""
-    sample_sequencing_metrics_details: list[str | int | float] = [
+    sample_sequencing_metrics_details: list[tuple] = [
```

Comment:
I think the previous typehint to be more informative

Reply:
but it was wrong. A completely explicit would be `list[tuple[str, str, int, float, float, int]`

Reply:
ok I will do `list[tuple[str | int | float]]`

Reply:
still wrong, I will leave it as it is

Reply:
I think just tuple